Dockerfile

```diff
@@ -271,8 +271,6 @@ RUN curl --connect-timeout 15 --retry 5 "https://storage.googleapis.com/flutter_
     ! -path /opt/dart/flutter \
     -delete
 
-USER root
```

Comment:
I am a not entirely sure if this change is desirable, I know it helped unblock some things for local development on your machine, but I think we want these files to be owned by root in production. I was able to build the image successfully from this branch without needing to make this change, even after clearing all my docker caches 🤔 

Could you try rebasing onto main and see if that resolves the issue? Otherwise I can dig into it and see if I can figure out what happened here.

Reply:
This commit is actually (at least that was the purpose) undoing an unwanted change from this pr. If you look at the global view (https://github.com/dependabot/dependabot-core/pull/4510/files) you see that this pr now neither adds or removes USER statements.

Reply:
Ah apologies I missed that 👍